authz/authz.go

```diff
@@ -112,6 +112,11 @@ p, *, *, *, /cas, *, *
 p, *, *, *, /api/webauthn, *, *
 p, *, *, GET, /api/get-release, *, *
 p, *, *, GET, /api/get-default-application, *, *
+p, *, *, POST, /api/enforce, *, *
+p, *, *, POST, /api/batch-enforce, *, *
+p, *, *, GET, /api/get-all-objects, *, *
+p, *, *, GET, /api/get-all-actions, *, *
+p, *, *, GET, /api/get-all-roles, *, *
```

Comment:
Now that the new api is allowed to be called, should we add comments to the Controller's functions to generate the corresponding Api documentation.